alphabase/constants/const_files/psm_reader.yaml

```diff
@@ -119,8 +124,7 @@ pfind:
     'uniprot_ids': 'Proteins'
     'fdr': 'Q-value'
     'decoy': ['Target/Decoy', 'Targe/Decoy']
-  modification_mapping:
-    '': ''
+  modification_mapping_type: 'maxquant'
```

Comment:
pFind does not use MaxQuant-like PTM format? Did this pass the pFind tests?

Reply:
good catch .. this works because in pFindReader, _translate_modifications() is overwritten .. will change the yaml